pkg/flow/internal/controller/loader.go

```diff
@@ -81,7 +83,9 @@ func (l *Loader) Apply(parentContext *hcl.EvalContext, blocks hcl.Blocks) hcl.Di
 		// We cache both arguments and exports during an initial load in case the
 		// component is new; we want to make sure that all fields are available
 		// before the component updates its exports for the first time.
-		l.evaluate(parentContext, n.(*ComponentNode), true, true)
+		if err := l.evaluate(parentContext, n.(*ComponentNode), true, true); err != nil {
+			stdlog.Fatal(err)
```

Comment:
We don't want this to always exit the process; this will get called every time the file gets loaded, even after the first load. 

Can this instead add an error-level diagnostic to report errors back to the caller? 

(Also, we already have a logger attached to the Loader which isn't stdlog)